packages/directory/addon/templates/components/dir-search-bar.hbs

```diff
@@ -0,0 +1,7 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{input
+    type='text'
+    class='dir-search-bar__input'
+    key-up=(action this.attrs.searchFor)
```

Comment:
This does a refreshmodel on every keyup.  are there performance implications to that?